src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -98,7 +103,9 @@ public EventStreamController(final EventTypeRepository eventTypeRepository, fina
     }
 
     @VisibleForTesting
-    List<NakadiCursor> getStreamingStart(final String topic, final String cursorsStr)
+    List<NakadiCursor> getStreamingStart(final TopicRepository topicRepository,
+                                         final String topic,
+                                         final String cursorsStr)
```

Comment:
In case of begin the very first timeline should be used, in case of cursors absence - the very last one